Source/WebCore/html/shadow/attachmentElementShadow.css

```diff
@@ -80,6 +80,25 @@ attachment#attachment-preview {
     overflow: hidden;
 }
 
+div#attachment-progress {
+    grid-row: 1;
+    grid-column: 1;
+    width: 100%;
+    aspect-ratio: 1;
+    border-radius: 50%;
+    color: -apple-system-secondary-label;
+    background: conic-gradient(currentcolor calc(var(--progress) * 100%), transparent 0);
+}
+
+/* FIXME: Combine border into attachment-progress above, when rdar://107621207 is fixed. */
+div#attachment-progress-circle {
```

Comment:
Would we also be able to remove this `div` entirely when rdar://107621207 is fixed. Other than that, it's not obvious to me why we need both.

Reply:
Oh, that's what I meant, move the border styling in the above div, and remove this div. I'll rephrase for clarity...